substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -755,19 +761,43 @@ impl CliConfiguration for PalletCmd {
 
 /// List the benchmarks available in the runtime, in a CSV friendly format.
 fn list_benchmark(
-	mut benchmarks_to_run: Vec<(
+	benchmarks_to_run: Vec<(
 		Vec<u8>,
 		Vec<u8>,
 		Vec<(BenchmarkParameter, u32, u32)>,
 		Vec<(String, String)>,
 	)>,
+	list_output: ListOutput,
+	no_csv_header: bool,
 ) {
-	// Sort and de-dub by pallet and function name.
-	benchmarks_to_run.sort_by(|(pa, sa, _, _), (pb, sb, _, _)| (pa, sa).cmp(&(pb, sb)));
-	benchmarks_to_run.dedup_by(|(pa, sa, _, _), (pb, sb, _, _)| (pa, sa) == (pb, sb));
```

Comment:
Uh, i think the calling code actually relies upon these being sorted by this function 🙈   
Can you maybe revive these two lines and put them in the parent function?

Reply:
hmm, but isn't it sorted and deduplicated by `BTreeMap` and `BTreeSet`?

Reply:
Oh 🤦 i thought the function took a `&mut` but it only took `mut` nvm 😆